controllers/volumereplicationgroup_controller.go

```diff
@@ -89,6 +89,7 @@ func (r *VolumeReplicationGroupReconciler) SetupWithManager(
 		}).
 		For(&ramendrv1alpha1.VolumeReplicationGroup{}).
 		Watches(&source.Kind{Type: &corev1.PersistentVolumeClaim{}}, pvcMapFun, builder.WithPredicates(pvcPredicate)).
+		Watches(&source.Kind{Type: &corev1.ConfigMap{}}, handler.EnqueueRequestsFromMapFunc(r.configMapFun)).
```

Comment:
@BenamarMk do we also need a predicate here like `builder.WithPredicates(predicate.ResourceVersionChangedPredicate{})` so that we act only on ConfigMap changes and not create/delete of the same?

Reply:
I guess the default predicate is fine for the configMap. We shouldn't find any VRGs on deletion or creation of the configMap.